lib/ecto/filters.ex

```diff
@@ -102,28 +93,19 @@ defmodule Ecto.Filters do
 
       defp get_filter_params(params) do
         cond do
-          is_list(params) and Keyword.has_key?(params, @key) -> Keyword.get(params, @key)
-          is_map(params) and Map.has_key?(params, @key) -> Map.get(params, @key)
-          is_map(params) and Map.has_key?(params, Atom.to_string(@key)) -> Map.get(params, Atom.to_string(@key))
-          true -> params
-        end
-      end
+          is_list(params) and Keyword.has_key?(params, @key) ->
+            Keyword.get(params, @key)
 
-      defp defaults({key, value}, query) do
-        source = elem(query.from.source, 1)
-        struct_keys = Map.keys(source.__struct__())
-
-        if Enum.member?(struct_keys, key) do
-          from(
-            [c] in query,
-            where: field(c, ^key) == ^value
-          )
-        else
-          query
+          is_map(params) and Map.has_key?(params, @key) ->
+            Map.get(params, @key)
+
+          is_map(params) and Map.has_key?(params, Atom.to_string(@key)) ->
+            Map.get(params, Atom.to_string(@key))
+
+          true ->
+            []
```

Comment:
Bit curious about this change since I wasn't able to figure out why this was changed from `params` to an empty list.

Reply:
before you could do `apply_filters(title: "test")` and it would work because params were returned here. I actually didn't realize that was possible until recently and I didn't want filters to get mixed up with other things you might configure from query_params like `page` or `page_size`.